ext/aws/FreeRTOSIPConfig.h.in

```diff
@@ -0,0 +1,293 @@
+/******************************************************************************
+ *
+ * MEMC CONFIDENTIAL
+ *
+ * Copyright (C) MEMC Electronic Materials SpA, 2020
+ *
+ * Unauthorized copying of this file, via any medium, is strictly
+ * prohibited.
+ * Proprietary and confidential.
```

Comment:
Where does this file come from?
The copyright notice does not sound like this file is legally usable in modm.

Reply:
[This official "default" config file](https://github.com/FreeRTOS/FreeRTOS/blob/master/FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/include/FreeRTOSIPConfigDefaults.h) is MIT licensed.

Reply:
Autsch, sorry, created a new file which gets automatically that header and then copied the remaining part. Will fix it.

Reply:
Fixed